sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/FileMetadataStructSuite.scala

```diff
@@ -622,4 +622,28 @@ class FileMetadataStructSuite extends QueryTest with SharedSparkSession {
       }
     }
   }
+
+  test(s"[SPARK-40918] Output cols around WSCG.isTooManyFields limit") {
```

Comment:
Nit: Can we change the string to "SPARK-40918: Output cols around WSCG.isTooManyFields limit"